sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/BarrierRequestHelperTest.java

```diff
@@ -178,7 +178,7 @@ public void barrierWithAadAuthorizationTokenProviderType() throws URISyntaxExcep
                 false,
                 null,
                 null,
-                new CosmosClientTelemetryConfig(),
+                new CosmosClientTelemetryConfig().sendClientTelemetryToService(false),
```

Comment:
should not this be false by default?

Reply:
when coming thourgh public API = yes. But some tests create RxDocumentSeviceClient manually - and there it is expected that the effecive flag is passed down. CosmosClinetBuilder is doing it when coming through public API.